app/models/manageiq/providers/vmware/cloud_manager/event_catcher/stream.rb

```diff
@@ -0,0 +1,105 @@
+require 'bunny'
+require 'thread'
+
+# Listens to RabbitMQ events
+class ManageIQ::Providers::Vmware::CloudManager::EventCatcher::Stream
```

Comment:
:+1: definitely like this here ... better to have it namespaced and in the same place as the rest of the vmware cloud manager stuff.